src/plugins/discourse/fetch.test.js

```diff
@@ -16,6 +16,12 @@ describe("plugins/discourse/fetch", () => {
         await snapshotFetcher().likesByUser("dl-proto", 0)
       ).toMatchSnapshot();
     });
+    it("loads user data from snapshot dl-proto", async () => {
+      expect(await snapshotFetcher().getUserData("dl-proto")).toMatchSnapshot();
+    });
+    it("loads user data from snapshot system", async () => {
+      expect(await snapshotFetcher().getUserData("system")).toMatchSnapshot();
+    });
```

Comment:
nit: just one is fine, i don't think there's much benefit from testing a second user